src/scripts/components/audio/Audio.jsx

```diff
@@ -49,6 +55,16 @@ export default class Audio extends React.Component {
     this.setState(this.baseState);
   }
 
+  changeColor(element, color) {
+    const { r, g, b, a } = color.rgb;
```

Comment:
Does this still work if a hex color is selected?

Reply:
Yes, the hex color still works even though the user inserts a hex code. Also works if an hsla code is updated :)